From 633ac404b73336ac430a8e786547bc7213db4667 Mon Sep 17 00:00:00 2001
From: Vitaly Andrianov <vitalya@ti.com>
Date: Tue, 31 May 2016 14:50:25 -0400
Subject: [PATCH 32/47] soc: ti: add missing include to knav_dma.h

The knav_dma.h uses "enum dma_transfer_direction" defined in dmaengine.h

Signed-off-by: Vitaly Andrianov <vitalya@ti.com>
---
 include/linux/soc/ti/knav_dma.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/soc/ti/knav_dma.h b/include/linux/soc/ti/knav_dma.h
index 0fc07b8..81a827e 100644
--- a/include/linux/soc/ti/knav_dma.h
+++ b/include/linux/soc/ti/knav_dma.h
@@ -17,6 +17,8 @@
 #ifndef __SOC_TI_KEYSTONE_NAVIGATOR_DMA_H__
 #define __SOC_TI_KEYSTONE_NAVIGATOR_DMA_H__
 
+#include <linux/dmaengine.h>
+
 /*
  * PKTDMA descriptor manipulation macros for host packet descriptor
  */
-- 
1.9.1

